textdb/textdb-common/src/test/java/edu/uci/ics/textdb/common/field/SpanTest.java

```diff
@@ -16,10 +16,16 @@ public void setUp(){
     
     @Test
     public void testGetters(){
+        String fieldName = "description";
         int start = 10;
         int end = 20;
-        span = new Span(start, end);
+        String key = "location";
+        String value = "new york";
+        span = new Span(fieldName, start, end, key, value);
         Assert.assertEquals(start, span.getStart());

```

Comment:
Just a thought: I can imagine there will be many test cases that need to compare two spans.  Is it possible to have a helper function to check the equality of two spans?